content/applications/inventory_and_mrp/barcode/setup/serial_numbers_lots.rst

```diff
@@ -0,0 +1,138 @@
+select the product by tapping or scanning a barcode and scan the lot or serial number for each
+quantity.
+
+.. tip::
+   Because vendor lot and serial number barcodes could be damaged in transit and arrive in a state
+   that cannot be scanned, it is a good practice to :ref:`set serial number or lot scanning to
+   optional <inventory/barcode/mandatory-scan>`. Otherwise, a broken barcode would block validation
+   of the receipt.
+
+For products that are starting out without a serial number, there are three options to manage them:
+manually typing in the number, generating them ahead of time, or disabling serial numbers for that
+operation.
+
+Manually input serial and lot numbers
+-------------------------------------
+
+For instances where there are few products or rare shipments, it may be easiest to type lot or
+serial numbers in at the time of receipt. After opening an existing receipt or creating a new one
+and scanning the product bar code, tap the :icon:`fa-pencil` :guilabel:`(pencil)` icon. Tap the
+:guilabel:`Serial/Lot Number` field and enter the serial number and tap :guilabel:`Confirm`.
+
+.. image:: serial_numbers_lots/manual-sn-input.png
+   :alt: Typing in a serial number.
+
+Generate serial numbers before a receipt
+----------------------------------------
+
+Lot and serial numbers can be generated from a scheduled receipt before the products arrive. This
+has the advantage of making it possible to print all the serial numbers with the picking order for
+the receipt.
+
+To generate lot and serial numbers, in the **Inventory** app click :guilabel:`Receipts` and select
+the receipt that requires generated lot or serial numbers. In the :guilabel:`Operations` tab, find
+the line for the product, and click the :icon:`fa-list` :guilabel:`(list)` icon. In the
+:guilabel:`Open: Stock move` pop-up menu, click :guilabel:`Generate Serials/Lots`.
+
+.. image:: serial_numbers_lots/serial-number-generate.png
+   :alt: Generate serial numbers for an incoming shipment.
+
+.. seealso::
+   :ref:`Assign serial numbers <inventory/product_management/assign-sn>`
+
+Disable lot and serial numbers for a receipt
+--------------------------------------------
+
+With lot and serial numbers disabled for an operation, products that are tracked by these methods
+can be received without supplying a serial number or lot number. It is still possible to scan a
+serial number at the receipt stage, but the missing numbers will need to be :ref:`created in the
+Inventory app <inventory/product_management/already-in-stock>`.
+
+To disable the use of serial numbers for a particular operation type, go to
+:menuselection:`Inventory app --> Configuration --> Operations Types` and select the operation type.
+In the :guilabel:`LOTS/SERIAL NUMBERS` section, untick :guilabel:`Create New` and :guilabel:`Use
+Existing ones`.
+
+.. image:: serial_numbers_lots/disable-serials-lots-for-op.png
+   :alt: Unchecked boxes next to 'Create New' and 'Use Existing ones'.
```

Comment:
Can remove this image as the written instruction is sufficient and there are a number of other images in this doc already
```suggestion

```